specifications/android.md

```diff
@@ -49,18 +55,21 @@ The exposed broadcast receiver of the distributor application MUST handle 2 diff
 
 ### org.unifiedpush.android.distributor.REGISTER
 
-The connector send this action to register to push messages. The intent MUST contain 2 String extras:
-* application: with the end user application package name. The distributor MUST be able to handle many connections with a single application.
-* token: with a random token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+The connector sends this action to register to push messages. The intent MUST contain 2 extras:
+* application (String): with the end user application package name. The distributor MUST be able to handle many connections with a single application.
+* token (String): with a random token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+
+It MAY be sent with following extra:
```

Comment:
Can we add another optional value?
* purpose (String): A short description of the purpose this registration fulfills which can be shown to the user by a distributor.

Reply:
I'm ok. I would just change "purpose" to "message" since it is quite the same meaning as "message" for registration failed.

* message (String) : A short description of the purpose of the registration that the connector MAY show to the user.

??